src/components/InstructorRevenueCard/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import {find, size, map} from 'lodash'
+import {Card} from 'egghead-ui'
+import {Text} from 'react-localize'
+import WrappedRequest from 'components/WrappedRequest'
+import LayoutColumns from 'components/LayoutColumns'
+import currentMonthStartDate from './utils/currentMonthStartDate'
+import totalRevenue from './utils/totalRevenue'
+import removeRevenueMonth from './utils/removeRevenueMonth'
+import RevenuePeriod from './components/RevenuePeriod'
+import LineChart from './components/LineChart'
+
+export default ({revenueUrl}) => revenueUrl
+  ? <WrappedRequest
+      url={revenueUrl}
+      subscribe
+    >
+      {({data}) => {
+        const currentMonthRevenue = find(data, ['month', currentMonthStartDate()])
+        const currentTotalRevenue = totalRevenue(removeRevenueMonth(data, currentMonthStartDate()))
+        const hasCurrentRevenue = size(data) > 0
+          && ((currentMonthRevenue && currentMonthRevenue.revenue > 0) || (currentTotalRevenue && currentTotalRevenue.revenue > 0))
+        const currentMonthDates = map(data, month => month.month)
+        const currentRevenuePoints = map(data, month => month.revenue)
+        const currentMinutesPoints = map(data, month => month.minutes_watched)
+        const revenueColor = '#4786ff' // # blue
+        const minutesColor = '#171e27' // # base-secondary
```

Comment:
Could remove these hard coded hex if this tachyons story is implemented:
https://github.com/eggheadio/tachyons-egghead/issues/28